data/json/recipes/armor/boots.json

```diff
@@ -16,6 +16,23 @@
     ]
   },
   {
+    "result": "boots_chitin",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 6,
```

Comment:
Backpack is just storage. Torso encumbering storage at that. At this point it doesn't mean much because you're highly likely to have a shopping cart or at least a wooden box to drag around.
Plus, it's a common zed drop.